### PR TITLE
Update for Year 3

### PR DESCRIPTION
**Pull request by kabilar**
- [x] Remove `Works` page since it says `Coming soon` and there are no immediate plans to add documentation.  Link directly to https://works.datajoint.com/.
- [x] Update layout for `Partnership` pages
- [x] Update Elements `Developer Guide` - Remove `jupytext` since we are no longer using
- [x] Update `Core` page - Add link to the `datajoint-tutorials` repo for GitHub Codespaces
- [x] Update `Core` page - Remove hyperlinks to improve readability
- [x] Update `Publications` page
  - [x] Add new publications, and remove duplicate items (i.e. bioRxiv) for manuscripts that have now been published
  - [x] Fix incorrect link
  - [x] Update html syntax to markdown syntax for external links
- [x] Move `Dissemination` plan to separate page, and update links
- [x] Simplify `Support` page
- [x] Update `Team` page
- [x] Update `Events` page
- [x] Add upcoming Elements
- [x] Remove Google Analytics environment variable since we are now using HubSpot

### CODE REVIEW

File: src/index.md [hunk 1]
```diff
@@ -38,7 +38,7 @@
 
      Managed services for neurophysiology built with DataJoint Core and DataJoint Elements.
 
-     [:octicons-arrow-right-24: Learn more](./works/)
+     <a href="https://works.datajoint.com/" target="_blank">:octicons-arrow-right-24: Sign in</a>
```
Comment (guzman-raphael) [inline src/index.md:41]:
```suggestion
     [:octicons-arrow-right-24: Sign in](https://works.datajoint.com/){:target="_blank"}
```

File: src/core/index.md [hunk 2]
```diff
@@ -12,23 +12,27 @@ make up the family of core open-source projects.
 
     ---
 
-    A low-level client for managing [data pipelines](./glossary#data-pipeline).
+    A low-level client for managing data pipelines.
+
+    <a href="https://github.com/datajoint/datajoint-tutorials" target="_blank">
+    :octicons-arrow-right-24: Interactive
+    tutorial on GitHub Codespaces</a>
```
Comment (guzman-raphael) [inline src/core/index.md:19]:
```suggestion
    [https://github.com/datajoint/datajoint-tutorials](:octicons-arrow-right-24: Interactive
    tutorial on GitHub Codespaces){:target="_blank"}
```

File: src/publications.md [hunk 3]
```diff
@@ -6,7 +6,15 @@ Title: Publications
 The following publications relied on DataJoint open-source software for data analysis.  If your work uses DataJoint or DataJoint Elements, please cite the respective
 [manuscripts and RRIDs](./citation.md).
 
-## 2022 <!-- 21 -->
+## 2023 <!-- 3 -->
```
Comment (guzman-raphael) [inline src/publications.md:9]: Maybe we should update the external links on this page as well to be more in MkDocs format.

**Review by guzman-raphael — CHANGES_REQUESTED** [review]
@kabilar Awesome, nice work. :+1:

File: src/core/index.md (hunk 2 above)
Comment (kabilar) [reply src/core/index.md:19]: Thanks!  Will update to the following:

```
[:octicons-arrow-right-24: Interactive tutorial on GitHub Codespaces](https://github.com/datajoint/datajoint-tutorials){:target="_blank"}
```

File: src/publications.md (hunk 3 above)
Comment (kabilar) [reply src/publications.md:9]: Updated!

**Comment by kabilar** [comment]
Thanks @guzman-raphael.  I have incorporated your suggestions.

File: src/community/partnership-nwb.md
Comment (kushalbakshi) [inline src/community/partnership-nwb.md:1]: Please update the person of contact in line 65

File: src/community/partnership-suite2p.md
Comment (kushalbakshi) [inline src/community/partnership-suite2p.md:1]: Please update the person of contact in line 65

**Review by kushalbakshi — COMMENTED** [review]
Just needs a few minor updates to contact information but other than that, it looks good.

**Comment by kabilar** [comment]
Thanks @kushalbakshi.  I have incorporated your suggestions.

**Review by kushalbakshi — APPROVED** [review]

**Comment by mkosma** [comment]
@kabilar sorry for the delay, I just came across the Github request for comments. Here's a few minor suggestions:

- Team: consider moving Tolga to the “past contributors” and whether others are appropriate to include (e.g., Raphael, Kushal, etc.)
- Dissemination: consider updating “tutorials on DataJoint Codebook” to reflect Codespaces approach (?)
- Publications: any new ones from the Tolias lab? Last week, Andreas said they’re using DataJoint on basically every project. (He's expecting to start working directly with us as he sets up his new lab at Stanford.)
- Consider renaming the “Adopt” segment to “Adoption — the parallel segments use nouns like “Selection” and “Dissemination”

**Comment by kabilar** [comment]
Page rename for `Adopt` to `Adoption`: https://github.com/datajoint-company/datajoint-docs/pull/47

**Comment by mkosma** [comment]
@kabilar thanks - and apologies, I must have reviewed in the wrong place. I had this message saying "review requested" but that brought me to this conversation thread, with a set of commits from 2 months ago at the top (i.e., not a PR with consolidated changes).

**Comment by kabilar** [comment]
> > * Publications: any new ones from the Tolias lab? Last week, Andreas said they’re using DataJoint on basically every project. (He's expecting to start working directly with us as he sets up his new lab at Stanford.)
> 
> I have email notifications for publications citing any of the DataJoint manuscripts and haven't received any recent notifications from the Tolias lab.

Correction, [Celii et al., bioRxiv 2023](https://datajoint.com/docs/publications/#2023) is from the Reimer lab and Tolias lab.

**Pull request merged**